packages/iTwinUI-react/src/core/Header/HeaderBasicButton.tsx

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { PolymorphicForwardRefComponent, useTheme } from '../utils';
+import { ButtonProps } from '../Buttons';
+
+export type HeaderBasicButtonProps = ButtonProps;
+
+type HeaderBasicButtonComponent = PolymorphicForwardRefComponent<
+  'button',
+  HeaderBasicButtonProps
+>;
+
+export const HeaderBasicButton: HeaderBasicButtonComponent = React.forwardRef(
+  (props, ref) => {
+    const {
+      onClick,
+      className,
+      children,
+      style,
+      type = 'button',
+      startIcon,
+      as: Element = 'button',
+      ...rest
+    } = props;
+
+    useTheme();
+
+    return (
+      <Element
+        className={cx('iui-header-breadcrumb-button', className)}
+        onClick={onClick}
+        ref={ref}
+        type={type}
+        style={style}
+        {...rest}
+      >
+        {startIcon &&
+          React.cloneElement(startIcon, {
+            className: startIcon.props.className,
+          })}
+        {children}
```

Comment:
In `Button` the children are included like: `{children && <span>{children}</span>}`. Does it not need to be written the same way? Especially for the check that `children` exists?

Reply:
Also applicable to line 55 in `HeaderDropdownButton` and line 68 in `HeaderSplitButton`.

Reply:
The children are defined in the `buttonProps` variable inside of `HeaderButton.tsx`, so they do exist and are already wrapped in a span with custom styling using `iui-header-breadcrumb-button-text`.